doc/install.rst

```diff
@@ -28,21 +28,26 @@ Ubuntu/Mint
 .. index:: Ubuntu Linux
 .. index:: Mint
 
-We maintain a `Ubuntu Repository <https://gerbera.jfrog.io/>`__.
+We maintain a `Ubuntu Repository <https://pkg.gerbera.io/>`__.
 
-To install the latest tagged release (>=1.8.0):
+To install the latest tagged release (>=2.1.0):
 
 .. code-block:: sh
 
-    curl -fsSL https://gerbera.jfrog.io/artifactory/api/gpg/key/public | sudo apt-key add -
-    sudo apt-add-repository https://gerbera.jfrog.io/artifactory/debian
+    wget -O - https://pkg.gerbera.io/public.asc 2>/dev/null | gpg --dearmor - | sudo tee /usr/share/keyrings/gerbera-keyring.gpg >/dev/null
+    echo "deb [signed-by=/usr/share/keyrings/gerbera-keyring.gpg] https://pkg.gerbera.io/debian/ $(lsb_release -c --short) main" | sudo tee /etc/apt/sources.list.d/gerbera.list >/dev/null
+    sudo apt-get update
```

Comment:
as [you mentioned](https://github.com/gerbera/gerbera/issues/2988#issuecomment-2048476571) the release isn't there yet:

``` console
Reading package lists... Done                                      
E: The repository 'https://pkg.gerbera.io/debian jammy Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```